dependencies.yaml

```diff
@@ -32,36 +32,64 @@ dependencies:
           - c-compiler
           - cxx-compiler
           - make
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              arch: x86_64
+              cuda: "11.8"
+            packages:
+              - nvcc_linux-64=11.8
+          - matrix:
+              arch: aarch64
+              cuda: "11.8"
+            packages:
+              - nvcc_linux-aarch64=11.8
```

Comment:
Here, you need matrix entries that match `cuda: "11.2"` and `cuda: "11.4"`.

Alternatively, you can move the `cuda: "12.0"` entry to the top and comment that all CUDA 11 versions should match the other case, like this example: https://github.com/rapidsai/cudf/blob/241257f31267efde2c82b8ee689d58a1ce9d1b47/dependencies.yaml#L445-L452

Reply:
```suggestion
          - matrix:
              arch: x86_64
              cuda: "11.2"
            packages:
              - nvcc_linux-64=11.2
          - matrix:
              arch: aarch64
              cuda: "11.2"
            packages:
              - nvcc_linux-aarch64=11.2
          - matrix:
              arch: x86_64
              cuda: "11.4"
            packages:
              - nvcc_linux-64=11.4
          - matrix:
              arch: aarch64
              cuda: "11.4"
            packages:
              - nvcc_linux-aarch64=11.4
          - matrix:
              arch: x86_64
              cuda: "11.8"
            packages:
              - nvcc_linux-64=11.8
          - matrix:
              arch: aarch64
              cuda: "11.8"
            packages:
              - nvcc_linux-aarch64=11.8
```